buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -621,6 +621,7 @@ class BeamModulePlugin implements Plugin<Project> {
         proto_google_cloud_spanner_v1: "com.google.api.grpc:proto-google-cloud-spanner-v1", // google_cloud_platform_libraries_bom sets version
         proto_google_cloud_spanner_admin_database_v1: "com.google.api.grpc:proto-google-cloud-spanner-admin-database-v1", // google_cloud_platform_libraries_bom sets version
         proto_google_common_protos                  : "com.google.api.grpc:proto-google-common-protos", // google_cloud_platform_libraries_bom sets version
+        pubsublite_beam_io                          : "com.google.cloud:pubsublite-beam-io:$google_cloud_pubsublite_version",
```

Comment:
It'd be better to put it next to all the other google-cloud clients:

https://github.com/apache/beam/blob/e1c1b8bde9c582cf7432e6475ad33a3a740ad8e9/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L535

Reply:
I'll change the variable name to "google_cloud_pubsublite_beam_io" so it can both remain alphabetical and be sorted up there.